src/main/java/duke/ToDo.java

```diff
@@ -0,0 +1,45 @@
+package duke;
+
+import java.time.LocalDate;
+
+public class ToDo extends Task{
```

Comment:
You may want to follow the naming convention of `ToDoTask` etc. like you did for `Command`